src/main/java/org/opentripplanner/common/model/extras/nihOptions/fields/Surface.java

```diff
@@ -8,12 +8,14 @@
 
     public static final NihOption optionName = NihOption.SURFACE;
 
-    public static final Surface GRAVEL = new Surface("Gravel");
-    public static final Surface CONCRETE = new Surface("Concrete");
-    public static final Surface EARTH = new Surface("Dirt/grass");
     // TODO: make new type for multiple choice options?
-    public static final Surface EARTH_CONCRETE = new Surface("Dirt/grass;Concrete");
+
+    // ordered in descending order of ease to traverse
+    public static final Surface CONCRETE = new Surface("Concrete");
     public static final Surface GRAVEL_CONCRETE = new Surface("Gravel;Concrete");
+    public static final Surface EARTH_CONCRETE = new Surface("Dirt/grass;Concrete");
+    public static final Surface GRAVEL = new Surface("Gravel");
+    public static final Surface EARTH = new Surface("Dirt/grass");
```

Comment:
I wonder if we should also assign a numeric difficulty rating of some sort to the Surface object. For example, boardwalks and pedestrian bridges often have non-concrete surfaces but are still easy to walk on.